tests/utils.py

```diff
@@ -24,14 +24,15 @@ def check_request_response(req, endpoint):
     assert req.status_code == endpoint.status_code
 
 
-def check_elasticsearch_transaction(elasticsearch, expected_count):
-    q = {'query': {'term': {'processor.name': 'transaction'}}}
+def check_elasticsearch_transaction(elasticsearch,
+                                    expected_count,
+                                    query={'query': {'term': {'processor.name': 'transaction'}}}):
```

Comment:
default arguments for functions/methods should never be mutable objects in Python, as they are initialized when the module is imported. e.g. imagine if you do `query['query'] = something_else` in the function body, you just changed the default for all following invocations of the function.

A better solution for this is:

    def check_elasticsearch_transaction(elasticsearch, expected_count, query=None):
        if query is None:
            query = {'query': {'term': {'processor.name': 'transaction'}}}